cmd/fs-v1.go

```diff
@@ -293,8 +285,8 @@ func (fs *FSObjects) crawlAndGetDataUsageInfo(ctx context.Context, endCh <-chan
 			return nil
 		}
 
-		// Get file size
-		fi, err := os.Stat(origPath)
+		// Get file size, no need to follow symlinks.
+		fi, err := os.Lstat(origPath)
```

Comment:
I noticed that FS can follow symlinks, that's why I did os.Stat() and not os.Lstat() to avoid breaking current deployements.

Reply:
On purpose @vadmeste we will not do that anymore since creating eloop scenario is easy and we waste resources crawling such files.